File: flowcontrols/decesionmaking/2ndlargenum.py
num1=int(input("enter the num1:"))
num2=int(input("enter the num2:"))
num3=int(input("enter the num3:"))
if(num1>num2 and num1>num3):
    if(num2>num3):
        print("2nd large is ",num2)
    else:
        print("2nd large is ",num3)
elif(num2>num1 and num2>num3):
    if (num1 > num3):
        print("2nd large is ",num1)
    else:
        print("2nd large is ",num3)
elif(num3>num2 and num3>num1):
    if (num1 > num2):
        print("2nd large is ",num1)
    else:
        print("2nd large is ",num2)
elif(num1==num2==num3):
    print("all num equal")

File: front_end_tech/html_parts/bootstrap/Javascript_prgm/flow_control/decesion_making/2nd_large_num.js
num1=Number(prompt('enter num1'))
num2=Number(prompt('enter num2'))
num3=Number(prompt('enter num3'))
if(num1>num2 && num1>num3){
    if(num2>num3){
        console.log("2nd large is ",num2)}
    else{
        console.log("2nd large is ",num3)}}
else if(num2>num1 && num2>num3){
    if (num1 > num3){
        console.log("2nd large is ",num1)}
    else{
        console.log("2nd large is ",num3)}}
else if(num3>num2 && num3>num1){
    if (num1 > num2){
        console.log("2nd large is ",num1)}
    else{
        console.log("2nd large is ",num2)}}
else if(num1===num2===num3){
    console.log("all num equal")}